site/docs/v1/tech/tutorials/integrate-dotnet-api.md

```diff
@@ -0,0 +1,191 @@
+
+If the API call is successful, the `identity` route will return the identity and claims of the user. If the API call is not successful, the `identity` route will return a `401 Unauthorized` response. In this case, the code will try refreshing the access token using the refresh token in the `app.rt` cookie. To do this, the code makes a `fetch` request to the [refresh token route](/docs/v1/tech/apis/hosted-backend#refresh) of the Hosted Backend API. If the refresh token is valid, FusionAuth's Hosted Backend API will return a new access token. The code then tries the `identity` route again with the new access token.
+
+
+{% include _callout-note.liquid content="A note on CORS (Cross-Origin Resource Sharing): For this setup to work, all components, including the web page, the .NET Core API, and FusionAuth, must be on the same domain. This is because the `app.idt` JWT cookie and `app.rt` refresh cookie are set to the same domain as the FusionAuth instance. Since everything runs on the same domain, CORS won't usually be an issue. However, you'll be running all of this on `localhost` to test, with each component running on a different port, which will cause CORS issues. For this reason, we need to enable CORS on FusionAuth for the login webpage to access the FusionAuth Hosted Backend API. To do this, navigate to 'Settings', then 'System' on the sidebar, and then the 'CORS' tab. Enable CORS, and check 'GET', 'POST', and 'OPTIONS'. Enable 'Allow Credentials', and  add `http://localhost:3000` to the list of allowed origins." %}
+
+{% include docs/_image.liquid src="/assets/img/docs/integrations/dotnet-hosted-api-integration/cors-settings.png" alt="CORS settings in FusionAuth for testing on Localhost" class="img-fluid bottom-cropped" width="1200" figure=false %}
+
+You can now serve up the login and profile static pages. An easy way to do this is to use the [http-server](https://www.npmjs.com/package/http-server) package via `npx`. Change to the `LoginPage` directory and run the following command:
+
+```shell
+npx http-server -a localhost -p 3000
+```
+
+This will serve the website on `http://localhost:3000`
+
+## Testing the API Flow
+
+Navigate to the login webpage at `http://localhost:3000`. Click the "Login" button. This will redirect you to the FusionAuth Hosted Backend API login page. Enter the email and password of the user created in the setup script earlier for your FusionAuth instance. You will be redirected to the login webpage, and the results from the `/me` FusionAuth endpoint alongside the "Call the API" button will be shown. Click this button to call the `identity` route of the {{page.technology}} API, which will echo back the identity and claims of the user, from the JWT in the `app.idt` cookie. You should see something similar to the following output:
```

Comment:
Using the same syntax mentioned above

```suggestion
Navigate to the login webpage at `http://localhost:3000`. Click the <span>Login</span>{:.uielement} button. This will redirect you to the FusionAuth Hosted Backend API login page. Enter the email and password of the user created in the setup script earlier for your FusionAuth instance. You will be redirected to the login webpage, and the results from the `/me` FusionAuth endpoint alongside the <span>Call the API</span>{:.uielement} button will be shown. Click this button to call the `identity` route of the {{page.technology}} API, which will echo back the identity and claims of the user, from the JWT in the `app.idt` cookie. You should see something similar to the following output:
```